src/Application/Asset/Result/StartUploadBatchResult.php

```
@@ -0,0 +1,21 @@
+
+final readonly class StartUploadBatchResult
+{
+    /**
+     * @param list<StartUploadBatchFileResult> $files
+     */
+    public function __construct(
+        public array $files,
+    ) {
+        foreach ($this->files as $i => $file) {
+            if (! $file instanceof StartUploadBatchFileResult) {
+                throw new \InvalidArgumentException(sprintf('files[%d] must be an instance of StartUploadBatchFileResult', $i));
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '17', column '27'). (undefined)

(MissingImport)